### PR TITLE
Feature/viewer

### PR DESCRIPTION
**Pull request by ssipasseuth**

### CODE REVIEW

**Comment by ssipasseuth** [comment]
The newly added debugger for qti creator requires https://github.com/oat-sa/extension-tao-itemqti/pull/444

File: manifest.php [hunk 1]
```diff
@@ -24,17 +23,26 @@
 	'label' => 'xmlEdit',
 	'description' => 'xml editing and debugging tools',
     'license' => 'GPL-2.0',
-    'version' => '0.1',
+    'version' => '0.2',
```
Comment (krampstudio) [inline manifest.php]: Please use version 0.2.0 instead

File: views/js/editor.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+                setValue(getValue());//ensure that the rendering is updated
+            },
+            hide : function hide(){
+                $container.hide().trigger('hide'+_ns);
+            }
+        };
+    }
+
+    return {
+        init : init
```
Comment (krampstudio) [inline views/js/editor.js]: is `{ 
   init : function(){ 
   }
}`
the format required by the hook ?

If yes, please document it, so no one will change the name...

File: views/js/hooks/qtiCreatorDebugger/hook.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    'xmlEdit/editor',
+    'tpl!xmlEdit/hooks/qtiCreatorDebugger/trigger',
+    'taoQtiItem/qtiCreator/helper/xmlRenderer'
+], function(_, $, xmlEditor, buttonTpl, xmlRenderer){
+    
+    'use strict';
+    
+    var _ns = '.qtiCreatorDebugger';
+    
+    var eventList = _.map([
```
Comment (krampstudio) [inline views/js/hooks/qtiCreatorDebugger/hook.js]: What's this list ?

File: views/js/editor.js (hunk 2 above)
Comment (ssipasseuth) [inline views/js/editor.js]: It is just the method returns by the editor and is not a hook on his own.
But https://github.com/oat-sa/extension-tao-xmledit/blob/feature/viewer/views/js/hooks/qtiCreatorDebugger/hook.js#L122-L124 and  https://github.com/oat-sa/extension-tao-xmledit/blob/feature/viewer/views/js/hooks/apipCreatorDebugger/hook.js#L71-L73 are hooks so I added the comment as suggested

**Comment by ssipasseuth** [comment]
Updated PR, ready for another review

**Pull request merged**